source/gui/settingsDialogs.py

```diff
@@ -2272,6 +2272,35 @@ def makeSettings(self, settingsSizer):
 			config.conf["virtualBuffers"]["autoFocusFocusableElements"]
 		)
 
+		# Translators: This is the label for a textfield in the
+		# browse mode settings panel.
+		textParagraphRegexLabelText = _("Regular expression for text paragraph navigation")
+		self.textParagraphRegexEdit = sHelper.addLabeledControl(
+			textParagraphRegexLabelText,
+			wxCtrlClass=wx.TextCtrl,
+			size=(self.scaleSize(300), -1),
+		)
+		self.textParagraphRegexEdit.SetValue(config.conf["virtualBuffers"]["textParagraphRegex"])
+		self.bindHelpEvent("BrowseModeSettingsTextParagraphRegexEdit ", self.textParagraphRegexEdit)

```

Comment:
Removes space at end.
Also, as per [userGuideStandards](https://github.com/nvaccess/nvda/blob/master/projectDocs/dev/userGuideStandards.md), the Feature Description Anchor should not include the settings category. Once the anchor is set it cannot be updated, while settings may move categories.

```suggestion
		self.bindHelpEvent("TextParagraphRegexEdit", self.textParagraphRegexEdit)
```